hypershift-operator/controllers/hostedcluster/internal/platform/platform.go

```diff
@@ -73,7 +74,7 @@ func GetPlatform(hcluster *hyperv1.HostedCluster, utilitiesImage string, pullSec
 	switch hcluster.Spec.Platform.Type {
 	case hyperv1.AWSPlatform:
 		if pullSecretBytes != nil {
-			capiImageProvider, err = imgUtil.GetPayloadImage(context.TODO(), hcluster, AWSCAPIProvider, pullSecretBytes)
+			capiImageProvider, err = imgUtil.GetPayloadImage(context.TODO(), &releaseinfo.RegistryClientProvider{}, hcluster, AWSCAPIProvider, pullSecretBytes)
```

Comment:
Although not relevant to your use case, can you modify this function to also take a release provider and pass that in?

Reply:
Done